node/nodeRunner.go

```diff
@@ -1124,15 +1125,47 @@ func cleanupStorageIfNecessary(workingDir string, cleanupStorage bool) error {
 	return nil
 }
 
-func copyConfigToStatsFolder(statsFolder string, configs []string) {
+func copyConfigToStatsFolder(statsFolder string, gasScheduleFolder string, configs []string) {
 	err := os.MkdirAll(statsFolder, os.ModePerm)
 	log.LogIfError(err)
 
+	err = copyDirectory(gasScheduleFolder, statsFolder)
+	log.LogIfError(err)
+
 	for _, configFile := range configs {
 		copySingleFile(statsFolder, configFile)
 	}
 }
 
+func copyDirectory(source string, destination string) error {
```

Comment:
Some unit tests would be great. Can add a TODO here and others can write the tests as this PR goes into a feature branch

Reply:
added todo